R/RunDiagnosticsCdm.R

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023 Observational Health Data Sciences and Informatics
+#
+# This file is part of CohortDiagnostics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' Run Diagnostics for a cohort set
+#' 
+#' This is the main function exported by the CohortDiagnositcs package. It runs the 
+#' diagnostic analysis and saves the results to a folder.
+#' 
+#'
+#' @param cdm A cdm_reference object create by CDMConnector::cdm_from_con()
+#' @param cohortSet A cohort set created by CDMConnector::readCohortSet()
+#' @param exportFolder The folder where the output will be exported to. If this folder does not exist it will be created.
+#' @param runAnalysis A numeric vector corresponding to the diagnostic analyses should be executed. See Details
+#' @param minCellCount  The minimum cell count for fields contains person counts or fractions.
+#' 
+#' @details runAnalysis argument
+#' 
+#' Pass a numeric vector matching the following analyses. Negative numbers will be interpreted as exclusions.
+#' Use a numeric vector to select the analyses you want to run. Negative numbers will exclude analyses.
+#' The selection works the same way as vector subsetting in R.
+#' 
+#' # Analyses to choose from:
```

Comment:
I am not against having a runAnalysis argument, but please fill in the descriptions of each analysis